crates/typst/src/symbols/sym.rs

```diff
@@ -495,6 +494,59 @@ pub(crate) const SYM: &[(&str, Symbol)] = symbols! {
         spade.stroked: '♤',
     ],
 
+    // Music.
+    note: [
+        up: '🎜',
+        down: '🎝',
+        whole: '𝅝',
+        half: '𝅗𝅥',
+        quarter: '𝅘𝅥',
+        quarter.alt: '♩',
+        eighth: '𝅘𝅥𝅮',
+        eighth.alt: '♪',
+        eighth.beamed: '♫',
```

Comment:
shouldn't this `beamed` also be `up` and `down`?

Reply:
These `*.beamed` symbols do not indicating the `up` or `down` in their (standard) unicode names. Naming them `up` or `down` can be over-interpretation.

Reply:
Nevermind, I confused beamed with ascend/descend